### PR TITLE
:sparkles: Add saddle points practice exercise

### PR DESCRIPTION
**Pull request by jonmcalder**

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
One tiny nit

File: exercises/practice/saddle-points/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```
[the remark this answers is not in the document]
Comment (jonmcalder) [reply exercises/practice/saddle-points/.meta/config.json:2]: Oops! Thanks!

**Comment by jonmcalder** [comment]
@glennj Could you check this please?

**Review by glennj — APPROVED** [review]
My R is a bit stale, but it looks good to me

**Pull request merged**